mtools/test/logfiles/sharding_379_mongos.log

```diff
@@ -0,0 +1,78 @@
+2020-01-28T14:26:53.776+1100 I CONTROL  [main] 
+2020-01-28T14:26:53.777+1100 I CONTROL  [main] ** NOTE: This is a development version (3.7.9) of MongoDB.
```

Comment:
@Giuliano-C Why is this test generated using an older development version of MongoDB?

We are only supporting stable/production releases (currently MongoDB 3.6, 4.0, and 4.2). If a test requires changes committed during the 3.7 development cycle, the test case should be the next production release that includes the change (so 4.0.0).